### PR TITLE
tfhub release blog post

### PR DESCRIPTION
**Pull request by dfalbel**

### CODE REVIEW

File: _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+install.packages("tfhub")
+```
+
+After installing the R package you need to install the TensorFlow Hub python package. You can do it by running:
+
+```{r}
+tfhub::install_tfhub()
+```
+
+> **Note**: If you are using the most recent CRAN release of reticulate (1.14.0), tfhub will be automatically installed when you `library` it for the first time.
```
Comment (javierluraschi) [inline _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd]: I would remove this comment, most people reading this are not going to have the latest `reticulate` version yet.

Once we bump the min-reticulate required version, we can change this post to remove `install_tfhub()`.

File: _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+## Simple example
```
Comment (javierluraschi) [inline _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd]: Maybe `Getting Started` instead?

File: _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+====================================================================
+Total params: 3,540,265
+Trainable params: 0
+Non-trainable params: 3,540,265
+____________________________________________________________________
+```
+
+This model can now be used to create predict Imagenet labels for an image. For example, let's see the results for the famous Grace Hopper's photo:
+
+![Grace Hopper](images/grace-hopper.jpg)
```
Comment (javierluraschi) [inline _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd]: The image looks huge, maybe do

```r
![Grace Hopper](images/grace-hopper.jpg){width=20%}
```

File: _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+```{r}
+img <- image_load("images/grace-hopper.jpg", target_size = c(224,224)) %>% 
+  image_to_array()
+img <- img/255
+dim(img) <- c(1, dim(img))
+pred <- predict(model, img)
+imagenet_decode_predictions(pred[,-1,drop=FALSE])
```
Comment (javierluraschi) [inline _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd]: Not sure why `imagenet_decode_predictions` returns a list, but think I rather do

```r
imagenet_decode_predictions(pred[,-1,drop=FALSE])[[1]]
```

To avoid printing `[[1]]` in the table.

File: _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+![TensorFlow Hub](images/tfhub.png)
+
+You can find more examples of `layer_hub` usage in the following articles of the TensorFlow for R website:
+
+- [Transfer Learning with tfhub](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification_with_tfhub/)
+- [Using tfhub with Keras](https://tensorflow.rstudio.com/guide/tfhub/hub-with-keras/)
+- [tfhub Basics](https://tensorflow.rstudio.com/guide/tfhub/intro/)
+- [Text classification example](https://tensorflow.rstudio.com/guide/tfhub/examples/text_classification/)
+
+## Recipes steps
```
Comment (javierluraschi) [inline _posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd]: I would rename this section to `Recipes and Specifications` and remove the `tfdatasets integration` heading, the tfdatsets section feels small enough that we should probably collapse it.

**Comment by dfalbel** [comment]
@skeydan Thanks very much for the review!
